designs/bee/src/cup.mjs

```diff
@@ -7,12 +7,29 @@ export const cup = {
   hideDependencies: true,
   after: neckTie,
   options: {
+    //Bee's
     topDepth: { pct: 54, min: 50, max: 80, menu: 'fit' },
     bottomCupDepth: { pct: 8, min: 0, max: 20, menu: 'fit' },
     sideDepth: { pct: 20.6, min: 0, max: 30, menu: 'fit' },
     sideCurve: { pct: 0, min: -50, max: 50, menu: 'fit' },
     frontCurve: { pct: 0, min: -50, max: 50, menu: 'fit' },
     bellaGuide: { bool: false, menu: 'fit' },
+    //changed from Bella
+    backArmholeCurvature: 0.63,
```

Comment:
Why does a bikini pattern concern itself with so many options about the back of the block it's based on?

Reply:
It doesn't that's why I locked the ones that do not affect Bee. This was present on v2 but was lost in the port. This is simply relocking them because they do not affect Bee.